pytensor/tensor/extra_ops.py

```diff
@@ -592,6 +593,10 @@ def squeeze(x, axis=None):
         # Nothing to do
         return _x
 
+    for i in axis:
```

Comment:
Better to do a single call to specify_broadcastable with all the axis at once (smaller generated graph)

Also can you add a comment explaining what's going on for future devs reading the source?